docs/cart/Handlers.html

```diff
@@ -0,0 +1,1432 @@
+<!DOCTYPE html>
```

Comment:
is this file a part of the front-end of the shopping cart or a replacement of swagger doc?

Reply:
No, this is documentation generated with jsdoc. I haven't seen anything about swagger documentation. Should I use that instead of jsdoc?

Reply:
Let's keep jsdoc for now...until swagger is resolved